lib/vanagon/platform/dsl.rb

```diff
@@ -43,6 +44,8 @@ def platform(name, &block)
                       Vanagon::Platform::Solaris10.new(@name)
                     when /^solaris-11/
                       Vanagon::Platform::Solaris11.new(@name)
+                    when /^win-/
```

Comment:
I think using the full string windows in the platform name will be clearer/cleaner.